### DISCUSSION TITLE
MIDI Translation/Filter/Converter onboard

### DISCUSSION THREAD

**Discussion by simonaglover95** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#49] [Feature Requests & Ideas]
Provide a facility for translating incoming CC messages into other messages and forward to appropriate ports.

Originally submitted on looped.in ideas page with 26 community votes

**Comment by fourth44** [comment]
Conversion to sysex would be awesome (#92), but the limitation is that there would be a 1-1 mapping between e.g. specific CC values to specific Sysex strings.
